src/modules/layouts/components/WebsiteLayout/Footer/Footer.jsx

```diff
@@ -39,29 +40,27 @@ const Footer = () => {
     [],
   );
 
+  const { error, response, submitForm } = useHubspotForm({
+    formGuid: 'd0e98a7b-aba5-412b-8c12-ace04835bad9',
+    pageName: '*Any Page (Footer)',
+    pageUri: 'https://colony.io/*',
+    portalId: '4846129',
```

Comment:
Shouldn't these values be passed in as an `env` variable at build time rather then hardcoding them in ?

Either via a `.env` file or just process variables: `PORTAL_ID=4846129 yarn build`

Reply:
They certainly *could* be. But this isn't exactly a privacy concern, as both `portalId` and `formGuid` are publicly available at HubSpot (all forms are public). What would you prefer?

Reply:
If they're public then they're fine as-is. :+1: